classes/webservice/.php_cs.cache

```diff
@@ -0,0 +1 @@
+{"php":"7.2.24-0ubuntu0.18.04.1","version":"2.15.1:v2.15.1#20064511ab796593a3990669eff5f5b535001f7c","indent":"    ","lineEnding":"\n","rules":{"blank_line_after_namespace":true,"braces":true,"class_definition":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_constants":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ensure_fully_multiline"},"no_break_comment":true,"no_closing_tag":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":true,"encoding":true,"full_opening_tag":true},"hashes":{"WebserviceRequest.php":3642012977}}
```

Comment:
Must be deleted

Reply:
ok, dear, The file has been deleted when I tried to squash my commits just worked for last 4 commits What is required of me